ext/standard/basic_functions.stub.php

```diff
@@ -262,8 +245,7 @@ function base64_decode(string $str, bool $strict = false): string|false {}
 
 /* basic_functions.c */
 
-/** @return mixed */
-function constant(string $name) {}
+function constant(string $name): mixed {}
```

Comment:
Theoretically, it's not exactly `mixed`... But I feel that it's ok to mark the return type as `mixed` rather than `bool|int|float|string|array|null`.

Reply:
It can also return resource ... the only thing it can't be is object, yet. `mixed` is definitely fine here.